src/app/public/modules/page-anchor/page-anchor.component.spec.ts

```diff
@@ -155,8 +155,10 @@ describe('StachePageAnchorComponent', () => {
     fixtureComponent.anchorContent = 'foo';
     fixture.detectChanges();
     expect(anchorComponent.name).toEqual('foo');
+
     fixtureComponent.anchorContent = 'bar';
-    expect(anchorComponent.name).toEqual = 'bar';
+    fixture.detectChanges();
+    expect(anchorComponent.name).toEqual('bar');
```

Comment:
Also causing the build to fail because it was using the wrong syntax for `toEqual`. Fixed to get the build to pass.

Reply:
😱